lib/bullet/detector/counter_cache.rb

```diff
@@ -21,11 +23,11 @@ def add_possible_objects(object_or_objects)
           return unless Bullet.counter_cache_enable?
 
           objects = Array.wrap(object_or_objects)
-          return if objects.map(&:bullet_primary_key_value).compact.empty?
+          return if objects.map { |object| object.bullet_primary_key_value }.compact.empty?
```

Comment:
@amatsuda what's the reason to make this change?

Reply:
It's described in the third commit https://github.com/flyerhzm/bullet/pull/709/commits/5a0286e0174ee898c032d28534547ec6cce8d8ca
We don't need this change if we could drop Ruby 2.3 support.

Reply:
@flyerhzm I decided to support ruby at least 2.7.0, please revert that commit.